python/cudf/tests/test_csvreader.py

```diff
@@ -630,3 +630,12 @@ def test_csv_reader_empty_dataframe():
     # should raise an error without dtypes
     with pytest.raises(GDFError):
         read_csv(StringIO(buffer))
+
+
+def test_csv_reader_filenotfound():
+
+    fname = "non-existing-filename.csv"
```

Comment:
Can you also test the other case you added, where the path exists but it is not a file (e.g. it's a directory)?

Reply:
Added test for passing in a directory